.github/ISSUE_TEMPLATE/regression.md

```diff
@@ -0,0 +1,63 @@
+---
+name: Regression
+about: Report a regression in Rust.
```

Comment:
```suggestion
about: Report a regression the Rust compiler or `rustdoc`.
```

Reply:
I think there's something missing in this suggestion

Reply:
I just went off of what was there for "Bug Report":

![image](https://user-images.githubusercontent.com/37223377/95520221-a0e63180-097b-11eb-8579-08f568da8e42.png)

Should it be changed?

Reply:
Oh hmm if that's what it is on the other issue template no need to change it here.